JavaScript/unit_testing/src/DOM.spec.js

```diff
@@ -0,0 +1,48 @@
+describe("DOM", function(){
```

Comment:
you are not testing the functionality
you have 5 test cases have you covered all the scenarios
all the possible cases 
most of the test cases you have check whether the spy has been invoked or not
you just checked the methods which are used in that method
but the main purpose of the method  should be tested
if i change the code and just place those methods which you are testing then the test case will pass
you are not testing the functionality

Could you write the test cases to test the main functionality